client/src/components/Review.jsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import axios from 'axios';
+import Style from './styles';
+import ReviewItem from './ReviewItem.jsx'
+import Paginations from './Paginations.jsx';
+import Ratings from './Ratings.jsx';
+
+export default class Review extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            reviews : [],
+            totalReviews : 0,
+            pageIndex : 1,
+            averageRatings : [],
+            keyword: '',
+            searchResultsReviewsTotal: 0,
+        };
+        this.getReviews = this.getReviews.bind(this);
+        this.renderReviews = this.renderReviews.bind(this);
+        this.searchKeyword = this.searchKeyword.bind(this);
+    }
+    componentDidMount() {
+        this.getReviews();
+    }
+    componentDidUpdate(prevProps) {
+        if (prevProps.locationId !== this.props.locationId) {
+            this.getReviews();
+        }
+    }
+    getReviews(pageIndex, keyword){
+        if (pageIndex == null) {   //checks for both null and undefined 
```

Comment:
you can simplify the line here by setting
pageIndex = pageIndex || this.state.pageIndex

Or the es6 way, you can set pageIndex to be default this.state.pageIndex in the parameters (line 32) ie pageIndex = this.state.pageIndex.

And same with keyword == null